Makefile

```diff
@@ -201,13 +208,35 @@ electron-installer-dmg: release/out/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-$
 TARGETS += \
 	electron-installer-dmg \
 	electron-installer-app-zip
+PUBLISH_AWS_S3 += \
+	release/out/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH).zip \
+	release/out/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH).dmg
 endif
```

Comment:
Now that the content of each conditional grew so much, maybe a blank line between `endif` and `ifeq` can help making things a bit easier to the eye?